discovery-provider/src/tasks/index.py

```diff
@@ -483,15 +502,15 @@ def update_task(self):
     # Define lock acquired boolean
     have_lock = False
     # Define redis lock object
-    update_lock = redis.lock("disc_prov_lock", timeout=25, blocking_timeout=25)
+    update_lock = redis.lock("disc_prov_lock", blocking_timeout=25)
```

Comment:
what happens in the case where a task takes > 25 sec now?

Reply:
per offline discussion - it just continues until execution has completed, ensuring other tasks that arise cannot acquire this lock